plugins/puppetdb/puppetdb

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/python3
+#
+# This code is licensed under GPLv3+
```

Comment:
Please use a perldoc formatted documentation instead. This will render the plugin nice in the gallery and for `munin-get`.

Reply:
I couldn't find any examples of this in other python plugins in the munin-contrib repository. can you point me to an example of what this would look like?

Reply:
ok nevermind this question. I found an example in the solr plugin. I've reformatted the top comment to make it a perldoc-style docstring. I'll send this in soon